source/documentation/tools/package-management.md

```diff
@@ -6,32 +6,100 @@ There are multiple package managers available for RStudio, JupyterLab & Airflow
 - Conda
 - packrat
 
-Renv is the current standard for Rstudio on the Analytical Platform as it provides simpler package management. 
+## Why use a package manager?
+
+This enables analysts to maintain a reproducible workflow by including a snapshot of all packages used within a project saved within the project files themselves that can be loaded and installed with a single consistent and reproducible method.
+
+This means that if you create some code one day, you (or another analyst who comes after you) should be able to pick it up several years later and run it without any difficulty - even if the packages used have themselves changed in the meantime.
+
+For Rstudio there is the added imperative to use a package manager (usually renv) because the analytical platform will remove installed packages when the docker image is restarted (which occurs automatically, roughly once a week).
```

Comment:
It's also a requirement to get apps working (docker requires a list of packages to install). Not sure we want to include this though, just making note of it.

Reply:
I don't think we should put apps in here - there should be specific guidance for that somewhere else I think

Reply:
If that already exists, then we could put a link to it here?  But otherwise I think this should be primarily an "RStudio" section of the guidance